ui/src/main/scripts/plugins/e2e/ts/e2e.ts

```diff
@@ -78,12 +79,15 @@ module E2E {
 
       _.each($scope.e2eData, (node) => {
         $scope.topLevel.push(node.id);
-        $scope.topLevel.sort();
         $scope.uris.push(node.uri);
         $scope.operations.push(node.operation);
       });
     };
 
+    $scope.compareNodes = function(a, b) {
+      return a.id > b.id;
```

Comment:
If this goes after https://github.com/hawkular/hawkular-apm/pull/584 it needs to be updated `a.serviceName + ':' + a.id ...`